src/ping.c

```diff
@@ -86,19 +112,32 @@ static pthread_mutex_t ping_lock = PTHREAD_MUTEX_INITIALIZER;
 static pthread_cond_t ping_cond = PTHREAD_COND_INITIALIZER;
 static int ping_thread_loop;
 static int ping_thread_error;
-static pthread_t ping_thread_id;
-
-static const char *config_keys[] = {"Host",    "SourceAddress", "AddressFamily",
-#ifdef HAVE_OPING_1_3
-                                    "Device",
-#endif
-                                    "Size",    "TTL",           "Interval",
-                                    "Timeout", "MaxMissed"};
-static int config_keys_num = STATIC_ARRAY_SIZE(config_keys);
 
 /*
  * Private functions
  */
+static void sourcelist_free() {
```

Comment:
Only C23 defaults to `()` being interpreted `(void)`, in earlier C-standards this is interpreted as K&R:
```suggestion
static void sourcelist_free(void) {
```